lib/index.js

```diff
@@ -29,6 +29,18 @@ class TokenTracker extends SafeEventEmitter {
       return this.createTokenFrom(tokenOpts)
     })
 
+    // initialize to empty array to ensure a tracker initialized
+    // with zero tokens doesn't emit an update until a token is added.
+    this._oldBalances = []
```

Comment:
This was broken in #33, but the situation in which it was broken wasn't covered by any tests.